src/api/notifications/NotificationAPISpec.js

```diff
@@ -108,6 +108,24 @@ describe('The Notifiation API', () => {
         });
     });
 
+    describe('the error method notificiation', () => {
```

Comment:
At some point we should add a test to see that if a new notification comes in while the notifications are being viewed, that they don't end up being minimized.